src/test/java/com/google/javascript/cl2dts/static_provided.js

```diff
@@ -0,0 +1,17 @@
+goog.provide('goog.events.BrowserEvent');
+goog.provide('goog.events.BrowserEvent.MouseButton');
+/**
+ * @constructor
+ */
+goog.events.BrowserEvent = function() {
+};
+
+/**
+ * Normalized button constants for the mouse.
+ * @enum {number}
+ */
+goog.events.BrowserEvent.MouseButton = {
+  LEFT: 0,
+  MIDDLE: 1,
+  RIGHT: 2
+};
```

Comment:
Since the test is called static_provided, what about adding a static function that is also provided (I think it should just work).